docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -24,52 +26,58 @@ In BODS, the following properties are used to represent such information disclos
 
 * ``beneficialOwnershipOrControl`` (See :ref:`schema-interest`)
 * ``directOrIndirect`` (See :ref:`schema-interest`)
-* ``componentStatementIDs`` (See :ref:`schema-relationship-record`)
+* ``componentRecords`` (See :ref:`schema-relationship-record`)
 * ``isComponent`` (See :ref:`schema-entity-record`)
 
 Requirements
 ------------------------
 
-The beneficial ownership relationship
+No beneficial ownership relationships
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If an entity declares that it has no disclosable beneficial owners, its record MUST be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty.reason`` 'noBeneficialOwners'.
+
+If an entity is exempt from declaring its beneficial owners its record SHOULD be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty.reason`` 'subjectExemptFromDisclosure'.
+
+
+A beneficial ownership relationship
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If a person is a beneficial owner of an entity, entity X, whether directly or indirectly, and one of them is required to declare this beneficial ownership:
 
-1. There MUST be a *primary* Ownership-or-control Statement connecting the two which represents the beneficial ownership relationship. In particular: 
+1. There MUST be a *primary* relationship connecting the two which represents the beneficial ownership relationship. Specifically, in the ``recordDetails`` for the relationship: 
 
-   a. the entity’s Statement MUST be the ``subject``;
-   b. the person’s Statement MUST be the ``interestedParty``;
+   a. the entity’s record MUST be the ``subject``;
+   b. the person’s record MUST be the ``interestedParty``;
    c. ``isComponent`` MUST be false;
-   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included in this primary Ownership-or-control Statement if known; and
-   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'.
-
-2. If beneficial ownership is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
+   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included if known; and
+   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'. If an interest is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
```

Comment:
```suggestion
   * if an interest is known to be exercised indirectly, via intermediary entities then ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
```